docker-compose.yml

```diff
@@ -16,45 +16,22 @@ services:
       - /var/lib/postgresql
 
   localstack:
-    image: localstack/localstack:latest
+    image: localstack/localstack:0.14.3
     ports:
-      - "9000:8080"
-      - "4572:4572"
-      - "4576:4576"
-      - "4575:4575"
+      - "4566:4566"
     networks:
-      - hocs-network
+      hocs-network:
+        aliases:
+          - localhost.localstack.cloud
     environment:
       HOSTNAME_EXTERNAL: localstack
       DEFAULT_REGION: eu-west-2
-      SERVICES: sns,sqs,s3
```

Comment:
I'd have thought this would be required as it effectively activates them within localstack.